snuba_sdk/visitors.py

```diff
@@ -264,6 +264,95 @@ def _visit_totals(self, totals: bool) -> str:
         return f"{totals}"
 
 
+class TranslationMQL(ExpressionVisitor[str]):
```

Comment:
If the only thing that's different about this is just a couple functions, then I would extend the existing `Translation` class and override just the changed functions.